db/schema.rb

```diff
@@ -491,6 +491,7 @@
     t.datetime "created_at", null: false
     t.integer "mentionable_id"
     t.string "mentionable_type"
+    t.bigint "organization_id"
```

Comment:
I believe this schema.rb needs to be regenerated. You'll notice it doesn't contain the new field.

Reply:
It looks `summary_html` was added to Users in #3269

Reply:
The `organization_id` shouldn't be here though

Reply:
> It looks summary_html was added to Users in #3269

Yeah, this is a problem, I've opened a PR to fix that https://github.com/thepracticaldev/dev.to/pull/3586

Reply:
I've rebased and force pushed the branch, to include the fix for this